.github/workflows/nodejs.yml

```diff
@@ -6,9 +6,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Use Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
```

Comment:
We need to add `permissions` to this workflow as well. If not, we'll get a security alert in the near future

Reply:
Just to confirm @dgreif is it the `read` permission?